plugins/adr-common/src/index.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Common types and functionalities for the ADR plugin.
+ * @packageDocumentation
+ */
+import { Entity, getEntitySourceLocation } from '@backstage/catalog-model';
+import { IndexableDocument } from '@backstage/plugin-search-common';
+
+/**
+ * ADR plugin annotation.
+ * @public
+ */
+export const ANNOTATION_ADR_LOCATION_DIR = 'adr/location-dir';
+
+/**
+ * Standard luxon DateTime format string for MADR dates.
+ * @public
+ */
+export const MADR_DATE_FORMAT = 'yyyy-MM-dd';
+
+/**
+ * Utility function to get the value of an entity ADR annotation.
+ * @public
+ */
+export const getAdrLocationDir = (entity: Entity) =>
```

Comment:
This doesn't seem to be used outside this package, perhaps keep it internal for now?